gips/datahandler/worker.py

```diff
@@ -226,3 +226,10 @@ def export_and_aggregate(job_id, start_ext, end_ext,
 
     if cleanup:
         shutil.rmtree(outdir)
+
+
+def look_busy(duration=5):
```

Comment:
I understand the usefulness of this for your testing, but should this be merged?